### DISCUSSION TITLE
[求救]注册了绳网打不开QAQ

### DISCUSSION THREAD

**Discussion by Lily960714** [share121/inter-knot#2272] [其他]
我只能看到一条条评论和图片,网站进去都说我没下载绳网小助手,我到底是少了哪一步QAQ

**Comment by Kamisato100** [comment]
所以你装了油猴脚本和绳网小助手了吗<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
